### PR TITLE
Remove CFP link and add post

### PR DESCRIPTION
**Pull request by lboynton**
Also adds next/prev links and share buttons to posts

### CODE REVIEW

**Comment by lboynton** [comment]
Closes #13 ?

File: _posts/2017-02-05-cfp-and-blind-bird-tickets-have-ended.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: CFP and blind bird ticket sales have ended
+layout: post
+---
+
+Thank you to everyone who has submitted a talk proposal for PHP South Coast 2017. We will now begin the difficult process of narrowing down the talk submissions to those we feel are best suited to the conference, aiming to have a balanced representation of both seasoned speakers and newcomers at the conference.
```
Comment (lboynton) [inline _posts/2017-02-05-cfp-and-blind-bird-tickets-have-ended.md:6]: Update as necessary

File: _includes/nav.html [hunk 2]
```diff
@@ -25,7 +25,7 @@
                         {% endif %}
                     {% endif %}
                 {% endfor %}
-                <li><a href="{{site.cfpUrl}}">Call for Papers</a></li>
+                <!--<li><a href="{{site.cfpUrl}}">Call for Papers</a></li>-->
```
Comment (asgrim) [inline _includes/nav.html:28]: we could've just deleted this ;) no need to keep it commented IMO

File: _posts/2017-02-05-cfp-and-blind-bird-tickets-have-ended.md (hunk 1 above)
Comment (asgrim) [reply _posts/2017-02-05-cfp-and-blind-bird-tickets-have-ended.md:6]: That's fine IMO :)

**Review by asgrim — APPROVED** [review]

File: _includes/nav.html (hunk 2 above)
Comment (lboynton) [reply _includes/nav.html:28]: It makes it easier when it gets copied and pasted next year ;)

**Pull request merged**